src/exports.jl

```diff
@@ -975,6 +981,7 @@ export maximal_normal_subgroups, has_maximal_normal_subgroups, set_maximal_norma
 export maximal_polyhedra
 export maximal_subgroup_reps
 export maximal_subgroups, has_maximal_subgroups, set_maximal_subgroups
+export min_revlex_basis_encoding
```

Comment:
This needs to move further down alphabetically.

Reply:
Done